backend/routes/announcements.js

```diff
@@ -29,19 +29,44 @@ const router = express.Router();
  *           description: The information displayed in the announcement card.
  *         createdAt:
  *           type: string
- *           format: date
+ *           format: YYYY-MM-DD
```

Comment:
DO we not care what time an announcement is made?

Reply:
Yup you're right I missed that